bson/json_util.py

```diff
@@ -824,22 +838,74 @@ def _parse_canonical_maxkey(doc: Any) -> MaxKey:
     return MaxKey()
 
 
+def _encode_bson(obj: Any, json_options: JSONOptions) -> Any:
```

Comment:
Can you inline this method? It's only called once and it should improve performance to remove a method call (per bson value).